Packs/CortexXDR/TestPlaybooks/Test_Playbook_-_Cortex_XDR_Malware_-_Incident_Enrichment.yml

```diff
@@ -0,0 +1,4565 @@
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "150"
+    scriptarguments:
+      agentsid:
+        simple: f8a2f58846b542579c12090652e79f3d
+      alertsandrelatedinfo:
+        simple: '[{"columnheader3":""},{},{}]'
+      externalsystemid:
+        simple: "63"
+      type:
+        simple: Malware Investigation and Response
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 590,
+          "y": -1010
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "150":
+    id: "150"
+    taskid: 837ed1d7-8847-42e3-8aae-58a611201e40
+    type: playbook
+    task:
+      id: 837ed1d7-8847-42e3-8aae-58a611201e40
+      version: -1
+      name: Cortex XDR Malware - Incident Enrichment
+      playbookName: Cortex XDR Malware - Incident Enrichment
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "84"
+    scriptarguments:
+      IncidentID:
+        complex:
+          root: incident
+          accessor: externalsystemid
+    separatecontext: false
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 590,
+          "y": -850
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "151":
+    id: "151"
+    taskid: 4f384f59-8318-4dd1-8c98-475e10ce4ea7
+    type: title
+    task:
+      id: 4f384f59-8318-4dd1-8c98-475e10ce4ea7
+      version: -1
+      name: Check Context Data
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "63"
+      - "67"
+      - "112"
+      - "39"
+      - "64"
+      - "69"
+      - "278"
+      - "65"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 590,
+          "y": 125
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "152":
+    id: "152"
+    taskid: 7bac4b5c-9141-4897-81db-cb2604f32666
+    type: condition
+    task:
+      id: 7bac4b5c-9141-4897-81db-cb2604f32666
+      version: -1
+      name: Verify MD5
+      description: |
+        Verify that the ‘MD5’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "140"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isExists
```

Comment:
I think we should change the `isExist` --> `isNotEmpty`
I am afraid that in case of a `null,` it will take the "verified" path instead of the "ELSE".
So if you see that it creates issues leave it as it is ( I want to be sure that we can handle another alert from xdr in case this specific will be deleted). If that works ok - maybe we should change that in the whole PB.